packages/main/src/MenuItem.js

```diff
@@ -56,7 +56,19 @@ const metadata = {
 		},
 
 		/**
-		 * Indicates if the any of the element siblings have children items.
```

Comment:
Indicates whether any of the element siblings have children items.

Reply:
Done